cdap-e2e-tests/src/e2e-test/features/controlcenter/ControlCenter.feature

```diff
@@ -0,0 +1,575 @@
+    Then Click on the Hamburger bar on the left panel
+    Then Click on Control Center link from the hamburger menu
+    Then Verify user is able to click on the dataset entity icon to see the details page of the dataset
+    Then Verify user is navigated to the details page of the dataset entity icon successfully
+
+  @BQ_INSERT_INT_SOURCE_TEST @BQ_SINK_TEST
+  Scenario: Verify that User is able to perform Delete operation on dataset entity successfully.
+    Given Open Datafusion Project to configure pipeline
+    Then Click on the Hamburger bar on the left panel
+    Then Click on Control Center link from the hamburger menu
+    Then Click on the Plus Green Button to import the pipelines
+    Then Verify user is able to click on the create button to create a pipeline successfully
+    When Expand Plugin group in the LHS plugins list: "Source"
+    When Select plugin: "BigQuery" from the plugins list as: "Source"
+    When Expand Plugin group in the LHS plugins list: "Sink"
+    When Select plugin: "BigQuery" from the plugins list as: "Sink"
+    Then Connect plugins: "BigQuery" and "BigQuery2" to establish connection
+    Then Navigate to the properties page of plugin: "BigQuery"
+    And Enter input plugin property: "referenceName" with value: "ReferenceDatasetControl"
+    And Replace input plugin property: "project" with value: "projectId"
+    And Enter input plugin property: "datasetProject" with value: "projectId"
+    And Replace input plugin property: "dataset" with value: "dataset"
+    Then Override Service account details if set in environment variables
+    And Enter input plugin property: "table" with value: "bqSourceTable"
+    Then Click on the Get Schema button
+    Then Validate "BigQuery" plugin properties
+    And Close the Plugin Properties page
+    Then Navigate to the properties page of plugin: "BigQuery2"
+    Then Replace input plugin property: "project" with value: "projectId"
+    Then Override Service account details if set in environment variables
+    Then Enter input plugin property: "datasetProject" with value: "projectId"
+    Then Enter input plugin property: "referenceName" with value: "BQReferenceNameDatasetControl"
+    Then Enter input plugin property: "dataset" with value: "dataset"
+    Then Enter input plugin property: "table" with value: "bqTargetTable"
+    Then Validate "BigQuery" plugin properties
+    And Close the Plugin Properties page
+    Then Save the pipeline
+    Then Deploy the pipeline
+    Then Run the Pipeline in Runtime
+    Then Wait till pipeline is in running state
+    Then Verify the pipeline status is "Succeeded"
+    Then Click on the Hamburger bar on the left panel
+    Then Click on Control Center link from the hamburger menu
+    Then Click on the delete icon of the dataset entity and the dataset should get deleted successfully
+    Then Verify the deleted dataset "BQReferenceNameDataset" entity is not present in control center page
```

Comment:
What is the dataset name we are searching for? What is the dataset name used?

Reply:
Changed the dataset name to the relevant one.